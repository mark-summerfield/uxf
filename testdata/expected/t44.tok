TClassBegin ttype=<IntPair>
  Field Str("First") vtype=int
  Field Str("Second") vtype=int
TClassEnd
TClassBegin ttype=<IntQuad>
  Field Str("First") vtype=IntPair
  Field Str("Second") vtype=IntPair
TClassEnd
ListBegin
  TableBegin ttype=<IntPair>
    Int(1)
    Int(2)
  TableEnd
  TableBegin ttype=<IntQuad>
    TableBegin ttype=<IntPair>
      Int(1)
      Int(2)
    TableEnd
    TableBegin ttype=<IntPair>
      Int(3)
      Int(4)
    TableEnd
  TableEnd
  TableBegin ttype=<IntQuad>
    TableBegin ttype=<IntPair>
      Int(1)
      Int(2)
    TableEnd
    TableBegin ttype=<IntPair>
      Int(3)
      Int(4)
    TableEnd
    TableBegin ttype=<IntPair>
      Int(5)
      Int(6)
    TableEnd
    TableBegin ttype=<IntPair>
      Int(7)
      Int(8)
    TableEnd
    TableBegin ttype=<IntPair>
      Int(9)
      Int(10)
    TableEnd
    TableBegin ttype=<IntPair>
      Int(11)
      Int(12)
    TableEnd
  TableEnd
ListEnd
Eof
----------------------------------------
