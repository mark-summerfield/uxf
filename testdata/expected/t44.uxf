uxf 1.0 TLM Config
=IntPair First:int Second:int
=IntQuad First:IntPair Second:IntPair
[
  (IntPair 1 2) 
  (IntQuad (IntPair 1 2) 
  (IntPair 3 4)) 
  (IntQuad
    (IntPair 1 2) 
    (IntPair 3 4)
    (IntPair 5 6) 
    (IntPair 7 8)
    (IntPair 9 10) 
    (IntPair 11 12)
  )
]
