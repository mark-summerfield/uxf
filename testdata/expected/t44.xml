<?xml version="1.0" ?>
<uxf version="1" custom="TLM Config">
  <ttypes>
    <ttype name="IntPair">
      <field name="First" vtype="int"/>
      <field name="Second" vtype="int"/>
    </ttype>
    <ttype name="IntQuad">
      <field name="First" vtype="IntPair"/>
      <field name="Second" vtype="IntPair"/>
    </ttype>
  </ttypes>
  <list>
    <table name="IntPair">
      <row>
        <int v="1"/>
        <int v="2"/>
      </row>
    </table>
    <table name="IntQuad">
      <row>
        <table name="IntPair">
          <row>
            <int v="1"/>
            <int v="2"/>
          </row>
        </table>
        <table name="IntPair">
          <row>
            <int v="3"/>
            <int v="4"/>
          </row>
        </table>
      </row>
    </table>
    <table name="IntQuad">
      <row>
        <table name="IntPair">
          <row>
            <int v="1"/>
            <int v="2"/>
          </row>
        </table>
        <table name="IntPair">
          <row>
            <int v="3"/>
            <int v="4"/>
          </row>
        </table>
      </row>
      <row>
        <table name="IntPair">
          <row>
            <int v="5"/>
            <int v="6"/>
          </row>
        </table>
        <table name="IntPair">
          <row>
            <int v="7"/>
            <int v="8"/>
          </row>
        </table>
      </row>
      <row>
        <table name="IntPair">
          <row>
            <int v="9"/>
            <int v="10"/>
          </row>
        </table>
        <table name="IntPair">
          <row>
            <int v="11"/>
            <int v="12"/>
          </row>
        </table>
      </row>
    </table>
  </list>
</uxf>
